docs/sources/writing-guide/topic-types/tutorial/index.md

```diff
@@ -96,6 +96,58 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
 
 If getting access to the tutorial data is complex, include the instructions in the steps of the tutorial. If getting access to the data is straightforward, include it in the "Before you begin" section.
 
+## Publish your tutorial
+
+Your tutorial source is stored in your project repo in a `tutorials` folder and mounted to the tutorials repo so that it can be displayed on the [Tutorials](/tutorials) page. The source is stored in your project repo to make it easy for team members to review and edit the content. 
+
+Follow the steps below to hide the tutorial from your project's table of contents and to display it on the Tutorials page. 
+
+### Hide your tutorial from your Table of Contents
+
```

Comment:
I would add an intro here that explains why I would want to hide the tutorial. Do we always want it hidden from the primary docs toc?